jiant/tasks/seq2seq.py

```diff
@@ -128,10 +137,33 @@ def update_metrics(self, logits, labels, tagmask=None, predictions=None):
             # Cut labels if predictions (without gold target) are shorter.
             labels = labels[:, : predictions.shape[1]]
             tagmask = tagmask[:, : predictions.shape[1]]
```

Comment:
Are we now creating this even in cases where we won't use it?

Reply:
Changed. Thanks!